app/controllers/mileage_reports_controller.rb

```diff
@@ -10,7 +10,7 @@ def index
     respond_to do |format|
       format.csv do
         send_data mileage_report.to_csv,
-          filename: "mileage-report-#{Time.zone.now.to_i}.csv"
+          filename: "mileage-report-#{Time.current.strftime("%Y-%m-%d")}.csv"
```

Comment:
If you have time, please add a test for this filename or open a github chore requesting a test for the filename. Thanks! :)

Reply:
Added the test in this PR itself :)